dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -92,24 +120,318 @@ public ContainerResource(final WebResource webResource,
+            .find(containerId, user, mode == PageMode.ANON)
+                : (mode.showLive) ? (Container) APILocator.getVersionableAPI()
+                    .findLiveVersion(containerShorty.longId, user, mode == PageMode.ANON)
+                        : (Container) APILocator.getVersionableAPI()
+                            .findWorkingVersion(containerShorty.longId, user, mode == PageMode.ANON);
+
+
+        APILocator.getPermissionAPI()
+            .checkPermission(contentlet, PermissionLevel.READ, user);
+        APILocator.getPermissionAPI()
+            .checkPermission(container, PermissionLevel.EDIT, user);
+
+        MultiTree mt = new MultiTree().setContainer(containerId)
+                .setContentlet(contentletId)
+                .setRelationType(uid);
+
+        
+        MultiTreeFactory.deleteMultiTree(mt);
+        
+        return Response.ok("ok")
+                .build();
+    }
+
+
+
+    @Path("/containerContent/{params:.*}")
+    public final Response containerContents(@Context final HttpServletRequest req, @Context final HttpServletResponse res,
+            @QueryParam("containerId") final String containerId, @QueryParam("contentInode") final String contentInode)
+            throws DotDataException, DotSecurityException, ParseErrorException, MethodInvocationException,
+            ResourceNotFoundException, IOException {
+
+        final InitDataObject initData = webResource.init(true, req, true);
+        final User user = initData.getUser();
+
+        Language id = WebAPILocator.getLanguageWebAPI()
+            .getLanguage(req);
+
+        PageMode mode = PageMode.get(req);
+
+        Container container = APILocator.getContainerAPI()
+            .find(containerId, user, mode == PageMode.ANON);
+
+        org.apache.velocity.context.Context context = VelocityUtil.getWebContext(req, res);
+        Contentlet contentlet = APILocator.getContentletAPI()
+            .find(contentInode, user, mode == PageMode.ANON);
+        ContainerStructure cStruct = APILocator.getContainerAPI()
+            .getContainerStructures(container)
+            .stream()
+            .filter(cs -> contentlet.getStructureInode()
+                .equals(cs.getStructureId()))
+            .findFirst()
+            .orElse(null);
+
+        StringWriter in = new StringWriter();
+        StringWriter out = new StringWriter();
+        in.append("#set ($contentletList")
+            .append(container.getIdentifier())
+            .append(" = [")
+            .append(contentlet.getIdentifier())
+            .append("] )")
+            .append("#set ($totalSize")
+            .append(container.getIdentifier())
+            .append("=")
+            .append("1")
+            .append(")")
+            .append("#parseContainer(\"")
+            .append(container.getIdentifier())
+            .append("\")");
+
+
+
+        VelocityUtil.getEngine()
+            .evaluate(context, out, this.getClass()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "evaluate"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)